### DISCUSSION TITLE
Upgrading Ant Media Server Cluster

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5356] [FAQs]
I have an Ant Media Server Cluster with multiple nodes (2 Origin and 4 Edge nodes) running with an older version 2.4.3
I made this cluster initially following the [manual cluster installation](https://antmedia.io/docs/guides/clustering-and-scaling/cluster-installation/) guide.
I want to upgrade my cluster nodes to the latest version of AMS.

How should I upgrade the instances and what process should I follow? Is there any specific point that I need to be careful about? 

Thank you!

**Comment by USAMAWIZARD** [comment]
IF you have installed the cluster manually here are the steps to update.

1. login to each server with ssh  
2. disconnect the server from the cluster
3. /usr/local/antmedia/change_server_mode.sh standalone
4. installing latest AMS
5. sudo wget https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh && sudo chmod 755 install_ant-media-server.sh
6. sudo nano ./install_ant-media-server.sh -r true
7.once the server is installed you can again connect the node back in the cluster.
8.  sudo nano ./install_ant-media-server.sh 
9. cd /usr/local/antmedia
sudo ./change_server_mode.sh cluster <MONGODB_SERVER_IP> <MONGODB_USERNAME> <MONGODB_PASSWORD>`
